packaging/nnstreamer.spec

```diff
@@ -102,6 +105,22 @@ Requires:	nnstreamer = %{version}-%{release}
 %description tensorflow-lite
 NNStreamer's tensor_fliter subplugin of TensorFlow Lite.
 
+%package python-filter
+Summary:  NNStreamer Python Custom Filter Support
+Requires: nnstreamer = %{version}-%{release}
+Requires: python
+Requires: python-numpy-devel
+%description python-filter
+NNStreamer's tensor_filter subplugin of Python.
+
+%package python-api
+Summary: NNStreamer Python API Support
+Requires: nnstreamer = %{version}-%{release}
+Requires: python
+Requires: python-numpy-devel
```

Comment:
Is there a reason to separate the two python packages?

Reply:
One for a python module which a user will import in a python script.
The other one for subplugin.